esp-hal/src/spi/master.rs

```diff
@@ -1454,8 +1455,6 @@ pub mod dma {
 
                     crate::dma::asynch::DmaTxFuture::new(&mut self.channel.tx).await;
 
-                    // FIXME: in the future we should use the peripheral DMA status registers to
```

Comment:
This comments are redundant, imo using dma then waiting for the last few bytes to get flushed is the optimal way.